controllers/openstackconfiggenerator.go

```diff
@@ -263,6 +264,18 @@ func (r *OpenStackConfigGeneratorReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, err
 	}
 
+	// add build-in heat environment files
+	var tripleoEnvironmentFiles []string
+	if instance.Spec.HeatEnvs != nil {
```

Comment:
with the above set default to an empty slice I think we won't require this.

Reply:
yes, avoiding the nil check is why I wanted to set a default